Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -198,9 +221,28 @@ public override long CountIndex(Guid chainId)
             int offset,
             int? limit)
         {
-            return IndexCollection(chainId)
-                .Find(Query.All(), offset, limit ?? int.MaxValue)
-                .Select(i => i.Hash);
+            int skip = 0;
+            int count = 0;
+            byte[] prefix = IndexKeyPrefix;
+
+            foreach (Iterator it in IterateDb(_chainDb, prefix, chainId))
```

Comment:
Is there reason to skip as offset manually, instead `IEnumerable.Skip()`?

Reply:
No, I'll fix it.